### ISSUE TITLE
 config path could not be located

### ISSUE THREAD

**Issue by voidplayer** [ValveSoftware/SteamVR-for-Linux#7]
* Steam client version (build number or date):  Feb 21 2017 at 20:24:31
* Distribution (e.g. Ubuntu): debian/testing
* Graphics driver version (run nvidia-settings): 275.27.10
* Gist for SteamVR System Information: https://gist.github.com/voidplayer/49f7f91632760ac98a179ac7626c27e5
* Opted into Steam client beta?: Yes
* Opted into SteamVR beta?: Yes
* Have you checked for system updates?: [Yes/No]

#### Please describe your issue in as much detail as possible:
When I start stamvr it shows: "steamVR failed to initalized for unknown reasons. (error: config path could not be located(111))"

The headset in the status goes green, but when I try to do the room set up, the headset is off

Dmesg shows:

```
[Wed Feb 22 20:58:11 2017] show_signal_msg: 87 callbacks suppressed
[Wed Feb 22 20:58:11 2017] vrcompositor[2138]: segfault at a8 ip 00007f1052885d42 sp 00007ffc37459878 error 4 in libnvidia-glcore.so.375.27.10[7f10514a3000+13ef000]
[Wed Feb 22 21:11:15 2017] vrmonitor[2082]: segfault at 1 ip 00007f4c5ec54018 sp 00007ffef02dfd80 error 4 in libudev.so.0.13.0[7f4c5ec50000+c000]
[Wed Feb 22 21:11:21 2017] vrcompositor[2658]: segfault at a8 ip 00007f22a07efd42 sp 00007fff1cb7ca08 error 4 in libnvidia-glcore.so.375.27.10[7f229f40d000+13ef000]
[Wed Feb 22 21:14:43 2017] vrmonitor[2602]: segfault at 7fbe9ac4add0 ip 00007fbd9d5dac44 sp 00007fff117cd450 error 4 in libQt5Core.so.5[7fbd9d31c000+533000]

```

**Comment by anoadragon453** [comment]
Can you try the latest Steam beta and see if the error persists? (Feb 23, 2017)

**Comment by voidplayer** [comment]
built: Feb 23 2017, at 00:38:08

same thing happens

**Comment by greyson** [comment]
I am getting the same error, however I have been able to run through Room Setup.  Of note, this may be related to why I am unable to enable direct mode (the configuration can't be found, so direct mode does not get enabled after the restart).

**Comment by ParkerMc** [comment]
Same on arch linux with my own driver and null driver.

**Comment by Kalman98** [comment]
I get the same thing as well, with ParkerMc's drivers, on Ubuntu 16.04.

**Comment by ParkerMc** [comment]
This is fixed for me with the update today.

**Comment by Kalman98** [comment]
Fixed for me as well.

**Comment by jessedubord** [comment]
I can also confirm the fix for me: Ubuntu 16.10 x64.
